src/engine/Collision/EdgeArea.ts

```diff
@@ -79,7 +79,7 @@ export class EdgeArea implements ICollisionArea {
    /**
     * Tests if a point is contained in this collision area
     */
-   public contains(point: Vector): boolean {
```

Comment:
It doesn't really matter, the implementation can leave off parameters from the interface. If you never intend to use point in this method, I'd prefer to leave it out